tests/test_wikipron/test_extract.py

```diff
@@ -2,7 +2,7 @@
 import requests_html
 
 from wikipron.extract import EXTRACTION_FUNCTIONS
-from wikipron.extract.core import _skip_pron
+from wikipron.extract.core import _skip_pron, _handle_parens
```

Comment:
Extremely minor but could you sort this lexicographically? thus `_handle_parens` first?

Reply:
sure! Also there's an addition to the README at line 78 you must've missed. Let me know if you'd like me to make any adjustments before merging

Reply:
Sorry I missed that. Thanks!